src/style.css

```diff
@@ -0,0 +1,91 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
```

Comment:
set these properties in `html` tag (add `font-family: inherit` for `.input` if it's needed)